tests/integration/external/guzzle7/test_no_cat_no_dt.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+Test that Cross Application Tracing (CAT) works with guzzle 7.
```

Comment:
This comment seems incorrect based on INI setting for CAT?

Reply:
@mfulb Funny thing - guzzle5 and guzzle6 tests have exactly the same problem :). I can update this comment in all 3 tests. Will this be more accurate description:
```suggestion
Test that neither CAT nor DT headers are added when CAT and DT are disabled.
```

Reply:
Done in 20a51012c057bb39c4bc303642c4136609eeb715